packages/xod-project/src/utils.js

```diff
@@ -102,6 +102,8 @@ export const generateId = shortid.generate;
  */
 export const validateId = R.test(/^[a-zA-Z0-9\-_]+$/);
 
+export const isGenericPinType = R.test(/^t[1-3]$/);
```

Comment:
Will shortening to `isGenericType` will introduce any confusion? I think `isGenericType` better reflects the essence.